lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -117,17 +118,25 @@ class IntegrationTestsManager {
   }
 
   Future<bool> prepareDriver() async {
+    final io.Directory priorCurrentDirectory = io.Directory.current;
     if (_driverDir.existsSync()) {
       _driverDir.deleteSync(recursive: true);
     }
 
     _driverDir.createSync(recursive: true);
 
+    // TODO(nurhan): We currently need git clone for getting the driver lock
+    // file. Remove this after making changes in web_installers.
     bool installWebInstallers = await _cloneWebInstallers();
     if (installWebInstallers) {
-      bool pubGet = await _runPubGet(pathlib.join(
-          _driverDir.path, 'web_installers', 'packages', 'web_drivers'));
-      if (pubGet) {
+      // Change the directory to the driver_lock.yaml file's directory.
+      io.Directory.current = pathlib.join(
+            _driverDir.path, 'web_installers', 'packages', 'web_drivers');
+      ChromeDriverInstaller chromeDriverInstaller = ChromeDriverInstaller();
```

Comment:
@yjbanov Please have a look. I started using ChromeDriverInstaller as a library. The package uses a fixed commit. I added the following dependency to pub:
```
  web_driver_installer:
    git:
      url: git://github.com/flutter/web_installers.git
      path: packages/web_drivers/
      ref: dae38d8839cc39f997fb4229f1382680b8758b4f
```

Reply:
Perfect!